server/dist/index.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
```

Comment:
I should probably just ignore anything in the DIST. 
This is basically compiled JS.